src/components/layouts/MainLayout.tsx

```diff
@@ -8,7 +8,7 @@ import { SideNavbar } from '@/components/navigation/navbar';
 
 const MainLayout = ({ children }: PropsWithChildren) => {
   return (
-    <div className='flex flex-col h-screen bg-heritage-blue-100'>
+    <div className='h-screen w-screen flex flex-col bg-heritage-blue-100 fixed'>
```

Comment:
hmm, the `fixed` here causes some scrolling issues, might need a fix before the next prod release

Reply:
sure, will be fixed with https://github.com/umccr/orca-ui/pull/79